src/guess/stringIndices.js

```diff
@@ -0,0 +1,10 @@
+function stringIndices( input , position ) {
+  let onlyLetters = input.replace(/[!@#$%^&*,]/g, ""),
```

Comment:
There's a character class for letters only that would be a little more straightforward than this regex.  As well, this would also capture whitespaces and numbers?